data/initrd/initrd.file_list

```diff
@@ -569,6 +570,10 @@ x etc/syslog.conf etc
 
 # additional scripts for linuxrc
 x scripts /
+# hmc exists only on s390
+if !exists(s390-tools-hmcdrvfs)
```

Comment:
It ultimately depends on the tools, not the arch. We can have the hmc url scheme once the hmcdrvfs command exists.